test/integration/smoke/test_activate_template.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
```

Comment:
Perhaps it's worth adding tests for: 
Register a template as systemvm template
Add tests for getSystemVMTemplateDefaultUrl - positive and negative
Investigate if upload template as system template is feasible/worth doing

Reply:
Already registering a template in the "activate" test.
Added a test for getSystemVMTemplateDefaultUrl.
Uploading a template will need to be done using a form post. This is theoretically possible but would take some effort.

Reply:
Please add at least a test for each new api and api change, we're now having 3 new apis and 3 old that have been changed